base_report_to_printer/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+from . import ir_actions_report
+from . import printing_action
+from . import printing_job
+from . import printing_printer
+from . import printing_server
```

Comment:
I think, it should be
```suggestion
from . import printing_server
from . import printing_printer
from . import printing_job
```